package.json

```diff
@@ -55,8 +55,7 @@
     "inflection": "^1.12.0",
     "jbox": "^1.0.8",
     "jquery": "^3.5.1",
-    "jquery-textcomplete": "^1.8.5",
-    "jquery-touchswipe": "^1.6.19",
+    "jquery-textcomplete": "^1.8.5",    "jquery-touchswipe": "^1.6.19",
```

Comment:
Not sure what happened here.

Reply:
Me neither... it's clear that I'm not at my best moment 🤸‍♂️